editor/editor_run.cpp

```diff
@@ -196,7 +197,7 @@ Error EditorRun::run(const String &p_scene, const String &p_custom_args, const L
 	for (List<String>::Element *E = args.front(); E; E = E->next()) {
 		printf(" %ls", E->get().c_str());
 	};
-	printf("\n");
+	putchar('\n');
```

Comment:
I'm not sure if this change is very useful, as `run()` isn't a method that is run often. Moreover, optimizing compilers may replace single-character `printf()` with `putchar()` on their own (see [this Stack Overflow question](https://stackoverflow.com/questions/23749797/putchar-vs-printf-is-there-a-difference)).

Reply:
Yes, the compiler can replace the `printf()` with a `putchar()`, but only in the optimization, and only if you use an adequated level optimization, witch makes the process slower anyway.